packages/ckeditor5-engine/tests/view/domconverter/view-to-dom.js

```diff
@@ -1232,6 +1232,24 @@ describe( 'DomConverter', () => {
 				sinon.assert.notCalled( warnStub );
 			} );
 
+			it( 'should yield the `RawElement` children properly', () => {
```

Comment:
Can we make this unit test a bit more complex, so there are at least a few children in the raw element (e.g. something like text, span, strong, text)?